var/spack/repos/builtin/packages/hypre-cmake/package.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+from spack import *
+
+
+class HypreCmake(CMakePackage, CudaPackage):
+    """Hypre is a library of high performance preconditioners that
+       features parallel multigrid methods for both structured and
+       unstructured grid problems."""
+
+    homepage = "http://computing.llnl.gov/project/linear_solvers/software.php"
+    url      = "https://github.com/hypre-space/hypre/archive/v2.14.0.tar.gz"
+    git      = "https://github.com/hypre-space/hypre.git"
+
+    maintainers = ['ulrikeyang', 'osborn9', 'balay']
+
+    test_requires_compiler = True
+
+    version('develop', branch='master')
+    version('2.22.0', sha256='2c786eb5d3e722d8d7b40254f138bef4565b2d4724041e56a8fa073bda5cfbb5')
+
+    variant('shared', default=(sys.platform != 'darwin'),
+            description="Build shared library (disables static library)")
+    variant('superlu-dist', default=False,
```

Comment:
```suggestion
    variant('superlu_dist', default=False,
```
See https://github.com/spack/spack/pull/24631#discussion_r662798100